www/src/pages/index.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import Menu from '../components/Menu';
 import Layout from '../components/PageLayout';
-import { Button } from '~paragon-react'; // eslint-disable-line
+import { Button, Container, Row, Col } from '~paragon-react'; // eslint-disable-line
```

Comment:
Were these just missing/it was broken? O_o

Reply:
I need to remove